server/core/store_test.go

```diff
@@ -95,6 +95,36 @@ func (s *testConcurrencySuite) TestCloneStore(c *C) {
 	wg.Wait()
 }
 
+func (s *testConcurrencySuite) TestShallowCloneStore(c *C) {
```

Comment:
What is the purpose of this test?

Reply:
I refer to the TestCloneStore test just to test the new ShallowClone method.

Reply:
`TestCloneStore` is used to test race problems for some create options like `SetStoreState`, you can take a look at https://github.com/pingcap/pd/pull/1802. I think there is no need to test it again? Or you can add a bench test for `ShallowClone` and `Clone` when executing `UpdateStoreStatus`, what do you think?

Reply:
Thanks.I remove the method of TestShallowCloneStore. I didn’t add bench test, because in performance, method ShallowClone is better than method Clone.  And This has been verified by the pd-heartbeat-bench test,  what do you think?